core-web/libs/sdk/react/src/lib/components/Container/Container.tsx

```diff
@@ -49,7 +53,12 @@ export function Container({ containerRef }: ContainerProps) {
           };
 
     const ContainerChildren = contentlets.map((contentlet) => {
-        const Component = components[contentlet.contentType] || NoContent;
+        const ContentTypeComponent = components[contentlet.contentType];
+        const DefaultComponent = components['Empty'] || NoComponent;
```

Comment:
```suggestion
        const DefaultComponent = components['XXXX'] || NoComponent;
```

Key `Empty` doesn't represent the intention here.

Reply:
maybe `CustomNoComponent` ?